docs/docs/50-user-guide/60-reference-docs/30-promotion-steps/json-parse.md

```diff
@@ -0,0 +1,77 @@
+---
+sidebar_label: json-parse
+description: Parses a JSON string and extracts values based on specified expressions.
+---
+
+# `json-parse`
+
+`json-parse` is a utility step that parses a JSON string and extracts values using [expr-lang][] expressions.
+
+## Configuration
+
+| Name | Type | Required | Description |
+|------|------|----------|-------------|
+| `path` | `string` | Y | Path to a JSON file. This path is relative to the temporary workspace that Kargo provisions for use by the promotion process. |
+| `outputs` | `[]object` | Y | A list of rules for extracting values from the parsed JSON. |
+| `outputs[].name` | `string` | Y | The name of the output variable. |
+| `outputs[].fromExpression` | `string` | Y | An [expr-lang](https://expr-lang.org/) expression that can extract the value from the JSON file. Note that this expression should not be offset by `${{` and `}}`. See examples for more details. |
+
+## Expressions
+
+The `fromExpression` field supports [expr-lang](https://expr-lang.org/) expressions.
+
+:::note
+Expressions should _not_ be offset by `${{` and `}}` to prevent pre-processing evaluation by Kargo. The `json_parser` step itself will evaluate these expressions.
```

Comment:
```suggestion
Expressions should _not_ be offset by `${{` and `}}` to prevent pre-processing evaluation by Kargo. The `json-parse` step itself will evaluate these expressions.
```